src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionXpathFilterTest.java

```diff
@@ -179,4 +180,79 @@ private static SuppressionXpathFilter createSuppressionXpathFilter(String fileNa
         return suppressionXpathFilter;
     }
 
+    private static LocalizedMessage getMessageForStringLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.PLUS, "", "",
+                null, null, "777", SuppressionXpathFilterTest.class, null);
+    }
+
+    private static LocalizedMessage getMessageForCharLiteral(int lineNo, int columnNo) {
+        return new LocalizedMessage(lineNo, columnNo, TokenTypes.CHAR_LITERAL, "", "",
+                null, null, "777", SuppressionXpathFilterTest.class, null);
+    }
+
+    private static TreeWalkerAuditEvent createTreeWalkerAudit(int lineNo, int columnNo,
```

Comment:
please move this method to bottom of the class, after all test methods. It is a common thing to have test cases at the top and all tech methods at the bottom.

Reply:
Done